driver-core/src/main/com/mongodb/event/ClusterClosedEvent.java

```diff
@@ -23,6 +23,8 @@
 /**
  * A cluster closed event.
  *
+ * <p>This event is synonymous to a TopologyClosedEvent</p>
```

Comment:
seems `synonymous with` is a little bit better, though not very sure. E.g. see https://expertiseinenglish.quora.com/https-www-quora-com-Which-is-correct-Synonymous-to-or-Synonymous-with-answer-Kailas-Linge